CHANGES.rdoc

```diff
@@ -4,6 +4,10 @@ This document describes the relevant changes between releases of the
 _hawkular-client_ project.
 
 
+=== v 3.0.3
+* Standardized Exceptions under the Hawkular namespace, and the old names were deprecated, here is the list:
+1. <code>Hawkular::BaseClient::HawkularException</code> -> <code>Hawkular::Exception</code>
+2. <code>Hawkular::BaseClient::HawkularConnectionException</code> -> <code>Hawkular::ConnectionException</code>
```

Comment:
the "signature" of the api changes because now another type of exception is returned, so it can be definitely considered as a breaking change.
I don't know if anyone uses it though.
@pilhuhn any idea?